scripts/ci_build_and_test_in_container.sh

```diff
@@ -153,6 +157,13 @@ if [[ "${RUN_INTEGRATED_TESTS}" = true ]]; then
   ATS_CMAKE_ARGS="-DATS_ARGUMENTS=\"--machine openmpi --ats openmpi_mpirun=/usr/bin/mpirun --ats openmpi_args=--allow-run-as-root --ats openmpi_procspernode=2 --ats openmpi_maxprocs=2\" -DPython3_ROOT_DIR=${ATS_PYTHON_HOME}"
 fi
 
+
+if [[ "${CODE_COVERAGE}" = true ]]; then
+  apt-get install -y lcov
```

Comment:
```suggestion
  or_die apt-get update
  or_die apt-get install -y lcov
```
You may need to update the cache of packages before you try to install your package.